pkg/apis/pipeline/v1alpha1/task_validation.go

```diff
@@ -95,9 +95,11 @@ func (ts *TaskSpec) Validate(ctx context.Context) *apis.FieldError {
 		}
 	}
 
-	if err := validateInputParameterVariables(ts.Steps, ts.Inputs); err != nil {
-		return err
-	}
+	/*
+		 	if err := validateInputParameterVariables(ts.Steps, ts.Inputs); err != nil {
+				return err
+			}
+	*/
```

Comment:
As said in the comments, this is commented to skip the "auto" migrate variable interpolation